src/main/java/bsh/security/SecurityGuard.java

```diff
@@ -0,0 +1,37 @@
+package bsh.security;
+
+/** It's the interface to implement a single SecurityGuard to be used by MainSecurityGuard */
+public interface SecurityGuard {
```

Comment:
Not sure if this needs to be public either.

Reply:
Yep, it really needs to be public, this interface must be implemented by users if them wanna prevent the execution of some specific thing, so them need to implement this interface just with their validation logic, then just add their implementation with `Interpreter.mainSecurityGuard.add()`

Reply:
Makes sense.